apps/io-services-cms-webapp/src/webservice/controllers/create-service.ts

```diff
@@ -59,8 +59,8 @@ type Dependencies = {
   fsmLifecycleClient: ServiceLifecycle.FsmClient;
   // An instance of APIM Client
   apimClient: ApiManagementClient;
-  // The APIM configuration
-  apimConfig: ApimConfig;
+  // Configuration
```

Comment:
nitpick: avoid to insert useless comment 😉 
```suggestion
```

Reply:
[0f02588](https://github.com/pagopa/io-services-cms/pull/115/commits/0f02588285efb043be4d3f79578e119405cc6eda)